src/loaders/ObjectLoader.js

```diff
@@ -225,15 +225,15 @@ THREE.ObjectLoader.prototype = {
 						break;
 
 					case 'TorusKnotGeometry':
+					case 'TorusKnotBufferGeometry':
 
-						geometry = new THREE.TorusKnotGeometry(
+						geometry = new THREE[ data.type ](
 							data.radius,
 							data.tube,
 							data.radialSegments,
 							data.tubularSegments,
```

Comment:
How should we handle this case? If we change `data.radialSegments` and `data.tubularSegments` things could break again, right?